src/data/currency_symbols.json

```diff
@@ -0,0 +1,174 @@
+{
+    "codes": [
```

Comment:
Is this intended to be used on genesis instead of the hard-coded vector, or is this file stale?

Reply:
I figured even if I didn't end up using it maybe in the future having a list of all currencies may play some role. But would try your suggestion of doing the conversions on chain spec for an overall cleaner approach.